content/_includes/contact.macros.njk

```diff
@@ -61,6 +62,11 @@ params:
     required=true
   ) }}
 
+  <label data-honey aria-hidden="true">
+    <span>Don't fill out this field:</span>
+    <input name="subject" tabindex="-1" />
```

Comment:
The `tabindex=-1` avoids people focusing on nothing as they `TAB` through the form. I hope that doesn't indicate to the bots that they should avoid this field 😅

Reply:
I hope so too!